awx/main/models/jobs.py

```diff
@@ -658,7 +658,7 @@ def notification_data(self, block=5):
                                           failures=h.failures,
                                           ok=h.ok,
                                           processed=h.processed,
-                                          skipped=h.skipped)
+                                          skipped=h.skipped)  # TODO: update with rescued, ignored
```

Comment:
Good find.  Can we track this via a bug/ticket elsewhere?

Reply:
Created https://github.com/ansible/awx/issues/4394, updated code to mention ticket 👍